test/faker/default/test_faker_bank.rb

```diff
@@ -73,481 +73,564 @@ def test_iban_ad
     assert_equal(24, account.length)
     assert_match(/^#{IBAN_HEADER}\d{2}[A-Z0-9]{18}$/, account)
+    assert iban_checksum(account), 'IBAN checksum is invalid'
   end
 
   # Sweden
   def test_iban_se
     account = @tester.iban(country_code: 'se')
     assert_equal(24, account.length)
     assert_match(/^#{IBAN_HEADER}\d{20}$/, account)
+    assert iban_checksum(account), 'IBAN checksum is invalid'
   end
 
   # Slovenia
   def test_iban_si
     account = @tester.iban(country_code: 'si')
     assert_equal(19, account.length)
     assert_match(/^#{IBAN_HEADER}\d{15}$/, account)
+    assert iban_checksum(account), 'IBAN checksum is invalid'
   end
 
   # Slovakia
   def test_iban_sk
     account = @tester.iban(country_code: 'sk')
     assert_equal(24, account.length)
     assert_match(/^#{IBAN_HEADER}\d{20}$/, account)
+    assert iban_checksum(account), 'IBAN checksum is invalid'
   end
 
   # San Marino
   def test_iban_sm
     account = @tester.iban(country_code: 'sm')
     assert_equal(27, account.length)
     assert_match(/^#{IBAN_HEADER}[A-Z]\d{10}[A-Z0-9]{12}$/, account)
+    assert iban_checksum(account), 'IBAN checksum is invalid'
   end
 
   # Timor-Leste
   def test_iban_tl
     account = @tester.iban(country_code: 'tl')
     assert_equal(23, account.length)
     assert_match(/^#{IBAN_HEADER}\d{19}$/, account)
+    assert iban_checksum(account), 'IBAN checksum is invalid'
   end
 
   # Tunisia
   def test_iban_tn
     account = @tester.iban(country_code: 'tn')
     assert_equal(24, account.length)
     assert_match(/^#{IBAN_HEADER}\d{20}$/, account)
+    assert iban_checksum(account), 'IBAN checksum is invalid'
   end
 
   # Turkey
   def test_iban_tr
     account = @tester.iban(country_code: 'tr')
     assert_equal(26, account.length)
     assert_match(/^#{IBAN_HEADER}\d{5}[A-Z0-9]{17}$/, account)
+    assert iban_checksum(account), 'IBAN checksum is invalid'
   end
 
   # Ukraine
   def test_iban_ua
     account = @tester.iban(country_code: 'ua')
     assert_equal(29, account.length)
     assert_match(/^#{IBAN_HEADER}\d{25}$/, account)
+    assert iban_checksum(account), 'IBAN checksum is invalid'
   end
 
   # Virgin Islands, British
   def test_iban_vg
     account = @tester.iban(country_code: 'vg')
     assert_equal(24, account.length)
     assert_match(/^#{IBAN_HEADER}[A-Z]{4}\d{16}$/, account)
+    assert iban_checksum(account), 'IBAN checksum is invalid'
   end
 
   # Kosovo, Republic of
   def test_iban_xk
     account = @tester.iban(country_code: 'xk')
     assert_equal(20, account.length)
     assert_match(/^#{IBAN_HEADER}\d{16}$/, account)
+    assert iban_checksum(account), 'IBAN checksum is invalid'
   end
 
   def test_iban_invalid
     assert_raise ArgumentError.new('Could not find iban details for bad') do
       @tester.iban(country_code: 'bad')
     end
   end
+
+  private
+
+  # https://en.wikipedia.org/wiki/International_Bank_Account_Number#Validating_the_IBAN
+  def iban_checksum(iban)
```

Comment:
Since this returns a bool, do you think renaming would make sense?
```suggestion
  def valid_iban_checksum?(iban)
```